modules/kotest/src/test/kotlin/org/testcontainers/containers/ContainerPerSpecListenerTest.kt

```diff
@@ -0,0 +1,27 @@
+import io.kotest.core.spec.style.StringSpec
+import io.mockk.mockk
+import io.mockk.verify
+import org.testcontainers.containers.ContainerPerSpecListener
+import org.testcontainers.containers.GenericContainer
+
+class ContainerPerSpecListenerTest : StringSpec() {
```

Comment:
already tested in the integration test. Also, we would prefer stubs over mocks

Reply:
Removed